woocommerce/woocommerce-gutenberg-products-block
# Add legacy hooks documentation

## What

Adds a document listing the hooks that exist in WC Core and in WC Blocks.

## Why

pdzGYk-1i5-p2#comment-1287



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Read the documentation and check for typos, grammar errors etc.
2. Check all the links work.


## Screenshots or screencast


N/A

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Skipping
